release-notes/major9/09.16.0/valtimo-backend-libraries.md

```diff
@@ -49,7 +49,13 @@ No new deprecations.
 
 ## Known issues
 
-* **Multiple form associations on process definition (36319)**
+* **Multiple existing form associations on process definition (36319)**
 
   When multiple start forms are associated with a single process definition the user is unable to start a new case of
   that specific type.
+  
+* **Using formAssociationService.createFormAssociation() on application startup causing the application to crash**
+
+  When form associations are created on application startup using the formAssociationService.createFormAssociation() method, these associations are duplicated. Before   the bugfix in this release these duplicated associations were added without errors. After the bugfix in this release the application will throw an exception when       creating an already existing association, causing the application to crash on startup.
```

Comment:
If we solved it by throwing an exception: is it still an issue, or has it become a feature?

Reply:
It is still an issue. There was no bugfix for this issue

Reply:
Aaa I guess the text is not entirely correct. The text mentions a bugfix which is unrelated to this one.Also I just looked it up the issue started a release 9.15.0. So this issue should be moved @ThomasRitense

Reply:
@Klaas-Ritense the text refers to the bugfix because the process_form_assocations_v2 table was introduced. Because that table has a unique constraint, duplicates are no longer possible and throw an error.

So @theo-ritense: this is a problem that implementations might have, thus an issue. But no bug, because implementations that have this issue are using the wrong method. 

@Klaas-Ritense I'll add it to the 9.15 known issues as well but wanted to include it here on the new documentation page.